pkg/quarterdeck/quarterdeck.go

```diff
@@ -271,16 +272,19 @@ func (s *Server) Routes(router *gin.Engine) (err error) {
 			projects.POST("/access", middleware.Authorize(perms.ReadTopics), s.ProjectAccess)
 		}
 
-		// Users Resource
+		// Users Resource - endpoint for Admin and Owner users of the organization
+		// to manager other users in their organization
 		users := v1.Group("/users", authenticate)
 		{
 			users.GET("/:id", middleware.Authorize(perms.ReadCollaborators), s.UserDetail)
 			users.PUT("/:id", middleware.Authorize(perms.EditCollaborators), s.UserUpdate)
 			users.GET("", middleware.Authorize(perms.ReadCollaborators), s.UserList)
 			users.DELETE("/:id", middleware.Authorize(perms.RemoveCollaborators), s.UserDelete)
+			users.POST("/:id", middleware.Authorize(perms.EditCollaborators), s.UserRoleUpdate)
```

Comment:
I remember discussing that we should have the role update be a POST, but feel free to correct me if I'm remembering wrong.